packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Custom/UnbrandedTypeSpecClient.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Demonstrates creating a client with an unbranded authentication token provider.
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+
+namespace UnbrandedTypeSpec
+{
+    /// <summary>
+    /// Demonstrates creating a client with an unbranded authentication token provider.
```

Comment:
```suggestion
    /// Demonstrates creating a custom client constructor to enable OAuth authentication.
```

Reply:
It looks like this wasn't updated.

Reply:
I think it has been updated:
![image](https://github.com/user-attachments/assets/a7ada378-3ee8-467c-9015-915b9d2b7e92)

Reply:
The comment at the top of the file is still talking about creating a client. That can be deleted.